meson.build

```diff
@@ -383,6 +383,10 @@ if host_os_family == 'linux'
   add_project_arguments('-D_GNU_SOURCE=1', language: c_languages)
 endif
 
+if host_os == 'tvos'
+  add_project_arguments('-I' + meson.current_source_dir() / 'src', language: c_languages)
+endif
```

Comment:
We should be able to avoid this if we add the header to `lib/base` -- everything internally depends on that being on the include path.